frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -342,6 +353,32 @@ export default {
       siretFormIsValid: true,
       duplicateSiretCanteen: null,
       siretDialog: false,
+      ministries: [
+        { value: "premier_ministre", text: "Service du premier ministre" },
+        { value: "affaires_etrangeres", text: "Ministère de l’Europe et des Affaires étrangères" },
+        { value: "ecologie", text: "Ministère de la Transition écologique" },
+        { value: "jeunese", text: "Ministère de l’Education nationale et de la Jeunesse et des Sports" },
+        { value: "economie", text: "Ministère de l’Economie, de la Finance et de la Relance" },
+        { value: "armee", text: "Ministère de l’Armée" },
+        { value: "interieur", text: "Ministère de l’Intérieur" },
+        { value: "travail", text: "Ministère Travail, de l’Emploi et de l’Insertion" },
+        { value: "outre_mer", text: "Ministère des Outre-mer" },
+        {
+          value: "territoires",
+          text: "Ministère de la Cohésion des territoires et des relations avec les collectivités territoriales",
```

Comment:
```suggestion
          text: "Ministère de la Cohésion des territoires et des Relations avec les collectivités territoriales",
```
https://www.cohesion-territoires.gouv.fr/

Reply:
Fix : https://github.com/betagouv/ma-cantine/pull/1273/commits/74db159fa3452d7e19c962d9646ffc238875f86f